Source/control.cpp

```diff
@@ -864,37 +864,37 @@ void DrawPanelBox(const Surface &out, SDL_Rect srcRect, Point targetPosition)
 	out.BlitFrom(*pBtmBuff, srcRect, targetPosition);
 }
 
-void DrawLifeFlaskUpper(const Surface &out)
+void DrawLifeFlaskUpper(const Surface &out, int playerId)

```

Comment:
Considering that this UI isn't multi user friendly I don't see the point in making it multi user capable?

Reply:
Right, I will convert the PR into draft, this is more like a POC to me. I am still creating a strategy to have the couch-coop working. I think I will be working on this until I have it fully working (including UI), then I will slice it into smaller manageable PRs.

Thanks for reviewing